### ISSUE TITLE
Jenkins hanging on develop branch

### ISSUE THREAD

**Issue by lesserwhirls** [Unidata/netcdf-java#710]
Running the latest `develop` branch, Jenkin has hung twice on the following:

~~~
ucar.nc2.ft.coverage.TestCoverageSubsetTime > test1Runtime1TimeOffset FAILED
    java.lang.AssertionError: Momentum_flux_u-component_surface_Mixed_intervals_Average
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.junit.Assert.assertNotNull(Assert.java:713)
        at ucar.nc2.ft.coverage.TestCoverageSubsetTime.test1Runtime1TimeOffset(TestCoverageSubsetTime.java:95)
~~~

**Comment by JohnLCaron** [comment]
Maybe fixed by PR #711 ?

**Comment by lesserwhirls** [comment]
It hung again last week, but is now working 👍
